recipes/boostdep/all/conanfile.py

```diff
@@ -0,0 +1,58 @@
+from conans import CMake, ConanFile, tools
+import os
+
+
+class BoostDepConan(ConanFile):
+    name = "boostdep"
+    settings = "os", "arch", "compiler", "build_type"
+    description = "A tool to create Boost module dependency reports"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/boostorg/boostdep"
+    license = "BSL-1.0"
+    topics = ("conan", "boostdep", "dependency", "tree")
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def requirements(self):
+        self.requires("boost/1.74.0")
```

Comment:
Should be probably bumped to 1.75.0 as well.

Reply:
```suggestion
        self.requires("boost/1.75.0")
```